docs/user/hbase/visibilities.rst

```diff
@@ -13,13 +13,13 @@ Setup and Configuration
 
 To configure HBase for visibility filtering follow the setup in the HBase Book under the
 `Visibility Labels <http://hbase.apache.org/book.html#hbase.visibility.labels>`__ section of
-the HBase book which includes enabling the HFile v3 format and visibility coprocessors in your hbase-site.xml:
```

Comment:
Does v3 matter anymore or is it so implicit that it doesn't matter?

Reply:
yeah 3 is the default now.